indexer/processor.js

```diff
@@ -780,7 +739,7 @@ class NinaProcessor {
         }
       }
 
-      if (releasePublicKey && blacklist.indexOf(releasePublicKey) === -1) {
+      if (releasePublicKey) {
```

Comment:
we will need to make sure that the releasePublicKey here is not in the restricted table

Reply:
updated with check against restricted table.